frontend/src/components/housing/enquiry/EnquiryIntakeForm.vue

```diff
@@ -45,7 +46,7 @@ const { getConfig } = storeToRefs(useConfigStore());
 const assignedActivityId: Ref<string | undefined> = ref(undefined);
 const editable: Ref<boolean> = ref(true);
 const formRef: Ref<InstanceType<typeof Form> | null> = ref(null);
-const initialFormValues: Ref<any | undefined> = ref(undefined);
+const isFormReset: Ref<boolean> = ref(false);
```

Comment:
This is never being used. It's set as the `v-if` on the form, but the value never changes. Same goes for the enquiry form.